### ISSUE TITLE
Issue: Plugin is more expensive than default SM native

### ISSUE THREAD

**Issue by Rushaway** [srcdslab/sm-plugin-LagReducer#3]
After making multiples bench, it seem this plugin get more expensive by creating forwards based on gameframe instead of hooking OnGameFrame (default SM native) and do your thing in it.

In conclusion this plugin should be deprecated and archived.
External plugins relaying on this should use OnGameFrame instead.

**Comment by maxime1907** [comment]
the purpose of this plugin is to use ongameframe and process a specific number of clients per frame to control the number Y clients times X plugins that optimizes things like PlayerVisibility which takes a lot of CPU load

**Comment by maxime1907** [comment]
yes it takes more that ongameframe but you should compare instead if we loose more by putting back ongameframe everywhere which will process X plugins times 64 clients

**Comment by Rushaway** [comment]
Detailled data example: 
https://github.com/srcdslab/sm-plugin-BossHUD/pull/27
